news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: single
+title:  "*Flying*  Edition 3"
+sidebar:
+  nav: news
+redirect_from: /news/latest/
+---
+## Open Source: #BetterTogether
+> This month's issue focuses Spinnaker's strong open source DNA. The OSS community embraces Spinnaker, as it builds a bridge between the collaboration and interoperability story at the core of Linux culture, and the powerful cloud economy that has leveraged it to transform software delivery. Efforts to build native Kubernetes integration highlight this unique strength.
+
+## [The New Age of the K8S Operator](https://www.armory.io/blog/introducing-spinnaker-operator-a-kubernetes-native-blueprint-for-success/)
+Spinnaker Operator, a Kubernetes operator for Spinnaker, has reached Beta. With it installed, you can use kubectl commands to install, deploy, and upgrade any version of Spinnaker through its lifecycle. Read the [doc](https://github.com/armory/spinnaker-operator/blob/release-0.3.x/README.md), try it out, and use the #kubernetes-operator channel in Spinnaker Slack to share your feedback.
+
+## [Join Spinnaker Gardening Days #CommunityHack Virtually on April 9-10](https://github.com/spinnaker-hackathon/gardening)
+As contributors and project newcomers alike gear up for the first Spinnaker Gardening Days, we're collecting and discussing [project ideas](https://github.com/spinnaker-hackathon/gardening/wiki/Project-Ideas). Socializing your ideas at this stage will help shape them into projects, so please visit the Wiki and join or create #gardening-idea Slack channels, and discuss with your peers. Alternatively, visit your favorite SIG's channel or meeting, and ask where you can jump in. Don't forget to [register](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111) to reserve your ticket!
```

Comment:
> to reserve your ticket

Ticket for what? This is a virtual event w/ no max capacity I thought.

Reply:
This is a CTA to get people to register for the event. There is limited capacity for the training track, and folks will only get the swag/prizes/lunch tickets if they register. So, to answer your question....ticket for the event.